ogb/lsc/mag240m.py

```diff
@@ -163,7 +183,8 @@ def save_test_submission(self, input_dict: Dict, dir_path: str, mode: str):
 
 
 if __name__ == '__main__':
-    dataset = MAG240MDataset()
+    dataset = MAG240MDataset('/home/user/yanbing/pyg/ogb/ogb/lsc/dataset')
+    data = dataset.to_pyg_hetero_data()
```

Comment:
Let's test this separately?

Reply:
`/home/user/yanbing/pyg/ogb/ogb/lsc/dataset` is the dev root, will remove it.